assets/js/atomic/blocks/product-elements/button/index.ts

```diff
@@ -32,6 +34,11 @@ const blockConfig: BlockConfiguration = {
 	attributes,
 	supports,
 	edit,
+	save,
+	styles: [
+		{ name: 'fill', label: 'Fill', isDefault: true },
+		{ name: 'outline', label: 'Outline' },
```

Comment:
I think we could translate labels here

Reply:
Nice catch! I have added translation for labels in [2cd7e66](https://github.com/woocommerce/woocommerce-blocks/pull/8781/commits/2cd7e661158b31ff25e6e9987f511a77ea5e4d95). Can you please have a quick look? 🙂